applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -1,8 +1,16 @@
 @(
     id: Int,
     isMobile: Boolean = false
-)
+)(implicit request: RequestHeader)
 @import conf.switches.Switches.CommercialSwitch
+@import mvt.CommercialGalleryBannerAds
+
+@additionalSizes = @{
+    if(CommercialGalleryBannerAds.isParticipating)
+        Seq("970,250", "970,350")
```

Comment:
There's no slot for 970,350. 

Participants should see one of the following: 300x250, 970,250 or 900,250 (we keep MPU's in as we can then make the bidding for slots available to high value MPU's still).